plugins/base/src/test/kotlin/translators/DefaultDescriptorToDocumentableTranslatorTest.kt

```diff
@@ -986,6 +986,30 @@ val soapXml = node("soap-env:Envelope", soapAttrs,
             }
         }
     }
+
+    @Test
+    fun `should add data modifier to data objects`() {
```

Comment:
I think it should be in `model.ClassesTest`

Reply:
It's a good idea to add a test to `mode.ClassesTest`, maybe it might be accidentally modified by one of the transformers or something, as model tests check `documentablesTransformationStage`

I will leave this test here as well, because I want to have tests specifically for parsing documentables before they are merged or transformed, to test the translator layer specifically

Reply:
Done

Reply:
> I will leave this test here as well, because I want to have tests specifically for parsing documentables before they are merged or transformed, to test the translator layer specifically

The problem is more general than specific only for data objects.
As for me, you can modify `ObjectTest` to check documentable before and after the transforming. (run the same action function at the different stages)